src/AndcultureCode.CSharp.Sitefinity.Testing/Extensions/DatetimeOffsetExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Shouldly;
+using System;
+
+namespace AndcultureCode.CSharp.Sitefinity.Testing.Extensions
+{
+    public static class DatetimeOffsetExtensions
```

Comment:
Suggestion: Since these are single line functions, consider converting to an expression body.